src/posix/platform/openthread-core-posix-config.h

```diff
@@ -64,6 +64,17 @@
 #define OPENTHREAD_CONFIG_LOG_OUTPUT OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_LOG_LEVEL
+ *
+ * Define the compile-time log level which is the lowest log level
+ * that can be set at run-time by `otLoggingSetLevel`.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_LOG_LEVEL
+#define OPENTHREAD_CONFIG_LOG_LEVEL OT_LOG_LEVEL_DEBG
+#endif
```

Comment:
This enables DEBG level by default on posix. We have seen that it can be very verbose and impact performance of the BR/router device. I understand that is useful to have this for debugging but not sure if we want to make it be the default.

Here is my suggestion: 
- We are using `OPENTHREAD_CONFIG_LOG_LEVEL_DYNAMIC_ENABLE` already under posix
- I suggest we also set `OPENTHREAD_CONFIG_LOG_LEVEL_INIT` for posix to `OT_LOG_LEVEL_INFO`

With this, a new device starting with the recommendded posix settings will have `DEBG` level logs included in the build but uses `LOG_INFO` (we can change to `LOG_DEBG` through dynamic log level APIs when debugging issues).

Reply:
Agree that the default log level should not be `DEBG`, but can use `log level` command to switch to `DEBG`.

Reply:
Agree, I missed that configuration. The default value of `OPENTHREAD_CONFIG_LOG_LEVEL_INIT` is `OT_LOG_LEVEL_CRIT`, I am keeping the default value.

Reply:
CRIT sgtm.